README.md

```diff
@@ -1 +1 @@
-# training-react
\ No newline at end of file
+# training-react
```

Comment:
Why this has change?

Reply:
Maybe because I added a new line at the end of the file

Reply:
Make no sense, this file should has no change